CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+- On our local devices we have chosen to use a combination of [Python 3](https://www.python.org), [Pip](https://pypi.org/project/pip/), [JQ](https://stedolan.github.io/jq/), [AWS CLI V2](https://aws.amazon.com/cli/) and other related developer tools. The specific tools utilized varies on a project-by-project basis. The Python portions of our source code can be adapted into another language with a supported AWS Lambda runtime.
+- Most of our ADX projects limit the usage of packages to [The Python Standard Library](https://docs.python.org/3.7/library/index.html). By default, Python provides a robust set of tools to access and manipulate popular data file formats, and we exploit these features to their fullest extent. Frequent packages we used when interacting with datasets include [os](https://docs.python.org/3.7/library/os.html), [urllib](https://docs.python.org/3.7/library/urllib.html), [json](https://docs.python.org/3.7/library/json.html) and [csv](https://docs.python.org/3.7/library/csv.html). We also utilize [Boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html), AWS's ADK for Python, which is automatically bundled into AWS Lambda's Python runtimes.
+- We are creating various AWS resources throughout the process of deploying and maintaining our ADX projects, including [Data Exchange](https://docs.aws.amazon.com/data-exchange/), [S3](https://docs.aws.amazon.com/s3/), [Lambda](https://docs.aws.amazon.com/lambda/) and [EventBridge](https://docs.aws.amazon.com/eventbridge/). The creation of most of the needed resources is streamlined by the [CloudFormation](https://docs.aws.amazon.com/cloudformation/) template included with this project, ([pre-processing-cfn.yaml](./pre-processing/pre-processing-cfn.yaml)).
+
+#### Directory Layout
+While the exact directory layout for one of our ADX projects can vary on a project-by-project basis, a typical presentation of our projects is as follows:
+```
+.
+├── pre-processing                  # Data provider side source code files and automation code
+│   ├── pre-processing-code         # Source code
+│   │   ├── lambda_function.py      # Code to interact with ADX for creating a dataset and revision
+│   │   └── source_data.py          # Code to acquire dataset and perform necessary data transformation
+│   └── pre-processing-cfn.yaml     # CloudFormation template to setup data provider automation
+├── dataset-description.md          # Dataset description that goes on the ADX listing
+├── product-description.md          # Product description that goes on the ADX listing
+├── init.sh                         # Initialization script to kick-off the automation
+├── README.md
+└── .gitignore
+```
+For more details on the technologies and directory layout used in our ADX products, please visit [Getting started with publishing a data product on AWS Data Exchange](https://github.com/rearc-data/publish-a-data-product-on-aws-data-exchange).
+
```

Comment:
I think linking to this repository is fine, but maybe the directory layout should live there.

Reply:
Completely removed directory layout section. Edited line 65 to be shorter, and placed it at the end of the "What tools are you using throughout your ADX Products?" section.